ts/src/pro/bybit.ts

```diff
@@ -946,6 +995,11 @@ export default class bybit extends bybitRest {
         const messageHash = 'orderbook' + ':' + symbol;
         this.orderbooks[symbol] = orderbook;
         client.resolve (orderbook, messageHash);
+        if (limit === '1') {
```

Comment:
@sc0Vu by doing this the user can't subscribe to orderbook+bidsAsks simultaneously, can't you save this in `this.bidsAsks` instead?